AirCasting/Services/RemoveOldMeasurementsService.swift

```diff
@@ -0,0 +1,53 @@
+// Created by Lunar on 26/05/2021.
+//
+
+import Foundation
+import CoreData
+
+
+final class RemoveOldMeasurementsService {
+    
+    enum Error: Swift.Error {
+        case sessionFetchFailed
+        case removingMeasurementsSurplusFailed
+        
+    }
+    private let twentyFourHoursMeasurementCount = 24 * 60
+    
+    func removeOldestMeasurements(context: NSManagedObjectContext, uuid: SessionUUID) {
```

Comment:
It would be nicer with the label `in:` `removeOldestMeasurements(context: context, uuid: id)` vs `removeOldestMeasurements(in: context, uuid: id)`